test/server/cards/02_SHD/units/MigsMayfeldTriggerman.spec.ts

```diff
@@ -0,0 +1,44 @@
+describe('Migs Mayfeld, Triggerman', function() {
+    integration(function(contextRef) {
+        describe('Migs Mayfeld\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['pillage', 'forced-surrender'],
+                        groundArena: [{ card: 'grogu#irresistible' }, { card: 'migs-mayfeld#triggerman' }],
+                        base: 'command-center'
+                    },
+                    player2: {
+                        hand: ['confiscate', 'daring-raid', 'krayt-dragon'],
+                        groundArena: [{ card: 'wampa', upgrades: ['shield'] }],
+                        spaceArena: [{ card: 'system-patrol-craft' }],
+                        base: 'chopper-base'
+                    }
+                });
+            });
+
+            it('can deal two damage to a unit or base after a card is discarded from hand', function () {
+                const { context } = contextRef;
+
+                // CASE 1: Can deal two damage to a unit or base after a card is discarded from hand
+                context.player1.clickCard(context.pillage);
```

Comment:
We have No Bargain implemented which would simplify the setup here a bit and make it a little easier to parse what's going on. Though if we add the third case for next round (see below comment) then it would be good to keep pillage and target the controlling player instead to confirm that the ability can trigger for either player

Reply:
I added No Bargain for the third discard case. Once we have Force Throw, I'll use that instead of Forced Surrender or Pillage so the test case looks cleaner.